services/service-core/src/auth/test/keycloak.validator.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Humanitech Supply Trail
+ *
+ * Copyright (c) Humanitech, Peter Rogov and Contributors
+ *
+ * Website: https://humanitech.net
+ * Repository: https://github.com/humanitech-net/supply-trail
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { userInputValidator } from '../keycloak.validator';
+import Joi from 'joi';
+const notValid = 'should not validate an invalid user input';
+const firstName = 'John';
+const lastName = 'Doe';
+const username = 'johndoe';
+
+describe('userInputValidator', () => {
+  const testCases = [
```

Comment:
good approach with an array of test cases and iterating through each!

Reply:
thank you